bigtable/client/client_options_test.cc

```diff
@@ -18,25 +18,50 @@
 
 TEST(ClientOptionsTest, ClientOptionsDefaultSettings) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  EXPECT_EQ("bigtable.googleapis.com", client_options_object.endpoint());
+  EXPECT_EQ("bigtable.googleapis.com", client_options_object.data_endpoint());
+  EXPECT_EQ("bigtableadmin.googleapis.com",
+            client_options_object.admin_endpoint());
   EXPECT_EQ(typeid(grpc::GoogleDefaultCredentials()),
             typeid(client_options_object.credentials()));
 }
 
-TEST(ClientOptionsTest, ClientOptionsCustomEndpoint) {
-  setenv("BIGTABLE_EMULATOR_HOST", "testendpoint.googleapis.com", 1);
+namespace {
+class ClientOptionsEmulatorTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    // TODO(#100) - setenv() is a Unix specific call ...
+    setenv("BIGTABLE_EMULATOR_HOST", "testendpoint.googleapis.com", 1);
+    previous_ = std::getenv("BIGTABLE_EMULATOR_HOST");
```

Comment:
Don't you want to call `getenv()` *before* you call `setenv()`?

Reply:
Oops.